### PR TITLE
prefixed the vars, added resizefs, since first post PXE task is alway…

### PR DESCRIPTION
**Pull request by kennethgillen**
…s resize the PXE created LVs and FS.

Feel free to test with infra_testpr. There's a snapshot ready for reverting-to (name of snap is self documenting).

### CODE REVIEW

File: ansible/roles/lvm-partition/tasks/main.yml [hunk 1]
```diff
@@ -3,21 +3,22 @@
...
 
 - name: storage | format
   filesystem:
-    fstype: "{{ lvfilesystem }}"
-    dev: /dev/{{ vgname }}/{{ lvname }}
+    fstype: "{{ lvm_lvfilesystem }}"
+    dev: /dev/{{ lvm_vgname }}/{{ lvm_lvname }}
+    resizefs: yes
 
 - name: storage | mount scratch
```
Comment (manics) [inline ansible/roles/lvm-partition/tasks/main.yml]: Since you're cleaning up this file, can you change the name to `storage | mount`?

**Comment by manics** [comment]
I've opened #12 which can be tested alongside this. Overall there should be no change since these are either variable renames, rearrangements of tasks, or variable changes which should not have an effect on existing systems.

**Comment by sbesson** [comment]
@manics: okay with the last changes?

File: ansible/roles/lvm-partition/tasks/main.yml [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+- name: storage | mount | create logical volume
   lvol:
-    vg: "{{ vgname }}"
-    lv: "{{ lvname }}"
-    size: "{{ lvsize }}"
-    opts: "{{ lvopts }}"
+    vg: "{{ lvm_vgname }}"
+    lv: "{{ lvm_lvname }}"
+    size: "{{ lvm_lvsize }}"
+    opts: "{{ lvm_lvopts }}"
```
Comment (sbesson) [inline ansible/roles/lvm-partition/tasks/main.yml]: While testing I realized `lvol` `opts` is a 2.0+ feature. Are you using development ansible or a beta version @kennethgillen @manics ?

Comment (manics) [inline ansible/roles/lvm-partition/tasks/main.yml]: development. Note `lvm_lvopts` will need another fix but I can rebase #21: https://github.com/openmicroscopy/infrastructure/pull/21/files#diff-ed13ce14137be8371d9a6e4bf1420595

**Pull request merged**